pages/404.js

```diff
@@ -55,7 +55,23 @@ export default function Custom404() {
 }
 
 export async function getStaticProps({ locale }) {
+  /* Place-holder Meta Data Props */
+  const meta = {
```

Comment:
Just a question, why do we have this if we are not using it? Maybe we should add it to the page?